docs/gateway/gcs.md

```diff
@@ -0,0 +1,60 @@
+# Minio GCS Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+Minio GCS Gateway adds Amazon S3 compatibility to Google Cloud Storage.
+
+## Run Minio Gateway for GCS
+### Create service account key for GCS and get the credentials file
+1. Go to the [API Console Credentials page](https://console.developers.google.com/project/_/apis/credentials).
+2. From the project drop-down at the top, select your project.
+3. On the Credentials page, select the __Create credentials__ drop-down, then select __Service account key__.
+4. From the __Service account__ drop-down, select __New service account__
+5. Fill up __Service account name__ and __Service account ID__
+6. For the __Role__, click the selec dropdown to choose __Storage__ -> __Storage Admin__ _(Full control of GCS resources)_
+7. Click the __Create__ button. This will download a credentials file to your desktop. Let's call this credentials.json
+
```

Comment:
Instead of adding steps to create accounts here, can we just say `Minio gateway mode needs project-id to get started.` Then link to the relevant document?